File: pygCartWeb/src/main/webapp/js/service/cartService.js
//服务层
app.service('cartService',function($http){
	    	
	//读取列表数据绑定到表单中
	this.findCartList=function(){
		return $http.get('cart/findCartList.do');		
	}
	
	
	//添加购物车
	this.addGoodsToCart=function(itemId , num){
		return $http.get('cart/addGoodsToCart.do?itemId='+itemId +"&num="+num);		
	}
	
});

File: pygCartInterface/src/main/java/com/pinyougou/cart/service/CartService.java
package com.pinyougou.cart.service;

import java.util.List;

import com.pinyougou.datapojo.Cart;

public interface CartService {
	
	public List<Cart> addGoodsToCart(List<Cart> cartList , Long itemId , Integer num);
	
	public List<Cart> addGoosToCarrRedis(List<Cart> cartList , String username);
	
	public List<Cart> findCartListFromRedis(String username);
	
	public List<Cart> megireCarList(List<Cart> cookieList , List<Cart> redisList);


}
